conf/messages.cy

```diff
@@ -459,59 +476,59 @@ chargeType.refundDescription = gan i chi ofyn am ad-daliad o ordaliad a wnaethoc
 chargeType.refundDescription.agent = oherwydd i’ch cleient ofyn am ad-daliad o ordaliad a wnaeth
 
 chargeType.POAInstalmentTitle = Rhandaliad taliad ar gyfrif
-chargeType.POAInstalLPITitle = ???
+chargeType.POAInstalLPITitle = Llog ar asesiad i ddiogelu
 
 chargeType.POAReturnDebitChargeTitle = Balans taliad ar gyfrif
 chargeType.POAReturnCreditChargeTitle = Ad-daliad taliad ar gyfrif
 
-chargeType.VatReturnPOALPITitle = ???
+chargeType.VatReturnPOALPITitle = Llog ar daliad ar gyfrif
```

Comment:
You haven't added any LPP translations but I can see we do have at least some of them translated on the spreadsheets. The spreadsheets have them translated as a full sentence with the "for the period...", which is not a direct match for the LPP titles, but by comparing a few of the translations you can trim out the "for the period..." text which leaves what we want.